Create README.md for the outbound oss training course extension (#70)
todogroup/ospo-career-path: Contribution-and-Creation/README.md

# OSPO 101 Training Extension Module 6 & 7
## Open Source Contribution and Creation

> This course series is an extension of modules 6 and 7 from the OSPO 101 training course and is based on the [content from the outbound open source guide](https://github.com/todogroup/outbound-oss). Before taking this course, we encourage students to go through the introductory courses on using and creating open source projects:

Reviewer: I'm not sure if positioning this course as an extension to modules 6 and 7 from the OSPO 101 works best. The modules 6 and 7 are much more focused on practices and operation of concrete open source contributions than at the management of the process. So the audience is different, as the OSPO 101 focused on the people involved in daily contributions, but the outbound guide focus on people managing the framework for doing open source at an organization.

So maybe it would be better to see it the other way around, the outbound guide as a high-level view of managing open source contributions on an organizational level, and the modules from the OSPO 101 as follow-up courses for more details regarding the day-to-day work of doing contributions.

That said, there is some overlap in content, especially in reasons and motivations for open source contributions. Modules 7 from the OSPO 101 has similar content here as also the first section of the outbound guide has.

As a prerequisite for the outbound guide I would see a basic understanding of open source (legal, community, business, technical) and an understanding of managing open source in an organization (for example as OSPO). Not sure, what the best courses are for this. Maybe also something from the OSPO 101? There are some references to material in the outbound guide which could also be seen as prerequisites.

* [OSPO 101 - Module 6: Understanding Upstream Open Source Projects](https://github.com/todogroup/ospo-career-path/blob/main/OSPO-101/module6/README.md)
* [OSPO 101 - Module 7: Open Source Creation](https://github.com/todogroup/ospo-career-path/tree/main/OSPO-101/module7)

***

### Program Overview

For people at organizations looking to enhance their open source and/or IT strategy (people shaping the process), delving into outbound open source – which involves contributing to existing projects or initiating new ones – can be a transformative experience. 

> This training course is designed to guide open source managers adn stakeholders within organizations through this journey.

Reviewer: ```suggestion
This training course is designed to guide open source managers and stakeholders within organizations through this journey.
```

This program is designed around three main thematic areas, which will be formatted into modules:

**MODULE 1 - COLLABORATION**

> * Transitioning from usage (also called consumption) to Contribution

Reviewer: I would add a point "Motivation for contribution" here.

> * Transitioning from usage (also called consumption) to Contribution

Author: ```suggestion
* Transitioning from usage (also called consumption) to Contribution
* Motivation for contribution
```

  * Diving Deep into starting your organization's journey in contributing to open source projects
  * Exploring various contribution models and procedures suitable for enterprises of all scales

>   * Managing and encouraging spare-time contributions

Reviewer: I would not talk about encouraging spare-time contributions. The goal of corporate outbound open source should be to make it possible and effective to contribute as part of the job. Spare-time contributions are something a company has to deal with but it should be up to the people doing them, if they want to do them or not, and it should never be part of a company strategy to rely on spare-time contributions.

>   * Managing and encouraging spare-time contributions

Author: ```suggestion
```

>   * Managing and encouraging spare-time contributions

Author: ```suggestion
```

    
**MODULE 2 - CREATION**

* Embarking on the organization's open source project releases
  * An in-depth look at the  lifecycle of open source projects.
  * Understanding the strategic advantages of launching open source initiatives for your organization.
  * Navigating through legal intricacies, including Intellectual Property (IP) and licensing considerations.
  * Mastering administrative and technical aspects, from repository management to selecting the right tools for your project.
  * 
**MODULE 3 - SUSTAINABILITY AND RESILIENCE**

* Growing and nurturing and keeping an open source project resilient

>    * Techniques and best practices for promoting the open source project that matters to your organization

Author: ```suggestion
```

>    * Building and managing vibrant communities around the open soruce projects that matter to your organization.

Reviewer: The outbound guide focuses more on process and policy and not so much on promotion or community building. There are already other guides for that. So maybe put the focus on the metrics part here.

>    * Building and managing vibrant communities around the open soruce projects that matter to your organization.

Author: ```suggestion
```

   * An introduction to open source software analytics and crucial open source metrics to gauge success and areas of improvement

> By the end of this training course, participants will be well-equipped with the knowledge and tools necessary to contribute to open source projects and kickstart and release open source projects, ensuring they align with their organization's strategic objectives and foster a culture of collaboration and innovation.

Reviewer: This sounds like it's targeted at people who are actively doing contributions themselves. This is a bit in conflict with the definition of the audience in the next section. It should be made clear that the training addresses open source managers not open source developers, and that it's about shaping the contribution process.

> By the end of this training course, participants will be well-equipped with the knowledge and tools necessary to contribute to open source projects and kickstart and release open source projects, ensuring they align with their organization's strategic objectives and foster a culture of collaboration and innovation.

Author: ```suggestion
By the end of this training course, participants will be well-equipped with the knowledge and tools necessary to manage open source projects and teams within the organization and manage releases of open source projects, ensuring they align with their organization's strategic objectives and foster a culture of collaboration and innovation.
```

Author: @cornelius could you please review the proposed text?
